tests/accidental_tests.ts

```diff
@@ -3,122 +3,123 @@
  * Copyright Mohit Muthanna 2010 <mohit@muthanna.com>
  */
 
-Vex.Flow.Test.Accidental = (function () {
-  function hasAccidental(note) {
-    return note.modifiers.reduce(function (hasAcc, modifier) {
+import { Factory } from '../src/factory';
+import { equal, ok, test, Vex, VF, QUnit } from './declarations';
+import { StaveNote } from '../src/stavenote';
+import { Accidental } from '../src/accidental';
+
+export const AccidentalTests = (function () {
+  function hasAccidental(note: any) {
+    return note.modifiers.reduce(function (hasAcc: boolean, modifier: any) {
       return hasAcc || modifier.getCategory() === 'accidentals';
     }, false);
   }
 
   // newAccid factory
-  function makeNewAccid(factory) {
-    return function (accidType) {
+  function makeNewAccid(factory: Factory) {
+    return function (accidType: any) {
       return factory.Accidental({ type: accidType });
     };
   }
 
-  var Accidental = {
+  return {
     Start: function () {
       QUnit.module('Accidental');
-      Vex.Flow.Test.runTests('Accidental Padding', Vex.Flow.Test.Accidental.formatAccidentalSpaces);
-      Vex.Flow.Test.runTests('Basic', Vex.Flow.Test.Accidental.basic);
-      Vex.Flow.Test.runTests('Stem Down', Vex.Flow.Test.Accidental.basicStemDown);
-      Vex.Flow.Test.runTests('Cautionary Accidental', Vex.Flow.Test.Accidental.cautionary);
-      Vex.Flow.Test.runTests('Accidental Arrangement Special Cases', Vex.Flow.Test.Accidental.specialCases);
-      Vex.Flow.Test.runTests('Multi Voice', Vex.Flow.Test.Accidental.multiVoice);
-      Vex.Flow.Test.runTests('Microtonal', Vex.Flow.Test.Accidental.microtonal);
-      Vex.Flow.Test.runTests('Microtonal (Iranian)', Vex.Flow.Test.Accidental.microtonal_iranian);
-      Vex.Flow.Test.runTests('Sagittal', Vex.Flow.Test.Accidental.sagittal);
-      test('Automatic Accidentals - Simple Tests', Vex.Flow.Test.Accidental.autoAccidentalWorking);
-      Vex.Flow.Test.runTests('Automatic Accidentals', Vex.Flow.Test.Accidental.automaticAccidentals0);
-      Vex.Flow.Test.runTests(
-        'Automatic Accidentals - C major scale in Ab',
-        Vex.Flow.Test.Accidental.automaticAccidentals1
-      );
-      Vex.Flow.Test.runTests(
-        'Automatic Accidentals - No Accidentals Necsesary',
-        Vex.Flow.Test.Accidental.automaticAccidentals2
-      );
+      Vex.Flow.Test.runTests('Accidental Padding', AccidentalTests.formatAccidentalSpaces);
```

Comment:
What is the difference between `Test` used on the first tests and this `Vex.Flow.Test.runtest`?